bin/create-scripts/block.js

```diff
@@ -1,18 +1,21 @@
+#!/usr/bin/env node
+
 /** @format */
 const path = require( 'path' );
 const webpack = require( 'webpack' );
 
 const __rootDir = path.resolve( __dirname, '../../' );
-const entryPath = path.resolve( process.argv[ 2 ] );
+const entryPath = path.resolve( __rootDir, process.argv[ 2 ] );
 const sourceDir = path.dirname( entryPath );
-const outputDir = path.join( sourceDir, 'build' );
+const outputDir = process.argv[ 3 ] ? process.argv[ 3 ] : path.join( sourceDir, 'build' );
```

Comment:
fine for now, but after this let's add option-parsing so this doesn't get out of control with other necessary CLI options

Reply:
Yep! 💯